.github/workflows/karma.yml

```diff
@@ -52,6 +52,7 @@ jobs:
           tunnelName: ${{ env.SAUCE_TUNNEL_ID }}
 
       - run: yarn sauce:ci
+      - run: ENABLE_EXPERIMENTAL_SIGNALS=1 yarn sauce:ci
```

Comment:
Historically we run these kinds of things in both synthetic and native shadow modes. If one of the 2 split karma batches is too much slower than the other, feel free to spread them across the 2.